utilities/EluvioLiveCli.js

```diff
@@ -824,7 +824,7 @@ const CmdAccountShow = async () => {
   console.log("Account Show\n");
 
   try {
-    await Init({ debugLogging: argv.verbose, asUrl: argv.as_url });
+    await Init({ debugLogging: false });
```

Comment:
did we intend to remove verbose and asUrl here?

Reply:
yep